lte/gateway/docker/docker-compose.yml

```diff
@@ -0,0 +1,232 @@
+---
+version: "3.7"
+
+# Standard logging for each service
+x-logging: &logging_anchor
+  driver: ${LOG_DRIVER}
+
+# Standard volumes mounted
+x-standard-volumes: &volumes_anchor
+  - ${CONFIGS_DEFAULT_VOLUME}:/etc/magma/
+  - ${CONFIGS_TEMPLATES_PATH}:/etc/magma/templates
+  - ${ROOTCA_PATH}:/var/opt/magma/certs/rootCA.pem
+  - ${CERTS_VOLUME}:/var/opt/magma/certs
+  - ${CONFIGS_OVERRIDE_VOLUME}:/var/opt/magma/configs
+  - ${SNOWFLAKE_PATH}:/etc/snowflake
+
+x-generic-service: &service
+  volumes: *volumes_anchor
+  logging: *logging_anchor
+  restart: always
+  network_mode: host
+
+# Generic python anchor to avoid repetition for agw python services
+x-agw-python-service: &pyservice
+  <<: *service
+  image: ${DOCKER_REGISTRY}gateway_python:${IMAGE_VERSION}
+
+# Generic anchor for lte c services
+x-lte-cservice: &ltecservice
+  <<: *service
+  image: ${DOCKER_REGISTRY}gateway_c:${IMAGE_VERSION}
+
+services:
+  magmad:
+    <<: *pyservice
+    container_name: magmad
+    environment:
+      DOCKER_REGISTRY: ${DOCKER_REGISTRY}
+      DOCKER_USERNAME: ${DOCKER_USERNAME}
+      DOCKER_PASSWORD: ${DOCKER_PASSWORD}
+    command: /usr/bin/env python3 -m magma.magmad.main
+  mobilityd:
+    <<: *pyservice
```

Comment:
Some services have a healthcheck, while others seemingly don't. How can we rationalize this?

Reply:
I added more health check for directoryd, policydb and pipelined. 
we will need to update the others as well.
this is tracked here in issue #9352